docs/utilities/data_classes.md

```diff
@@ -81,6 +82,45 @@ Event Source | Data_class
     The examples provided below are far from exhaustive - the data classes themselves are designed to provide a form of
     documentation inherently (via autocompletion, types and docstrings).
 
+### API Gateway Authorizer V2
```

Comment:
Could you add docs for the other Authorizer (v1)? That will make the next release faster, or else I can do it later

Reply:
done